src/hdmf/common/resources.py

```diff
@@ -2,9 +2,11 @@
 import numpy as np
 from . import register_class, EXP_NAMESPACE
 from . import get_type_map
-from ..container import Table, Row, Container, AbstractContainer, ExternalResourcesManager
+from ..container import Table, Row, Container, AbstractContainer, Data, ExternalResourcesManager
+from ..data_utils import DataIO
 from ..utils import docval, popargs, AllowPositional
 from ..build import TypeMap
+from ..term_set import TermSet
 from glob import glob
 import os
 import zipfile
```

Comment:
Not critical for this PR, but it would be nice to add `_repr_html_` function similar to #883 for TermSet as well to be able to render term sets nicely in the docs and Jupyter notebooks. Similarly, it would be nice to create a `__repr__` to make a nice plain-text rendering for TermSets.

Reply:
Sorry, this comment applies to TermSet below, I'll close this thread and move it to the correct place.